assets/js/components/spectre/Checkbox.jsx

```diff
@@ -4,18 +4,19 @@ import classNames from 'classnames'
 import { InputSizes } from './Util'
 
 const Checkbox = (props) => (
-    <label className={classNames('form-checkbox', {
-        [`input-${props.size}`]: props.size !== undefined,
-    })}>
-        <input type="checkbox" name={props.name} checked={props.checked} />
-        {props.children}
-    </label>
+  <label className={classNames('form-checkbox', {
+    [`input-${props.size}`]: props.size !== undefined,
+  })}>
+    <input type="checkbox" name={props.name} checked={props.checked} />
+    {props.children}
+  </label>
 )
 
 Checkbox.propTypes = {
-    name: PropTypes.string,
-    checked: PropTypes.bool,
-    size: PropTypes.oneOf(InputSizes),
+  name: PropTypes.string,
+  checked: PropTypes.bool,
+  size: PropTypes.oneOf(InputSizes),
+  children: PropTypes.node,
```

Comment:
is it normal/necessary to define this? every component has `props.children`

Reply:
Just-in-case.